src/status_im/subs/settings.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im.subs.settings
+  (:require [re-frame.core :as rf]))
+
+(rf/reg-sub
+ :settings/change-password-current-step
+ :<- [:settings/change-password]
+ (fn [change-password]
```

Comment:
it could be just keyword, because it's a function , so 
```clojure
(rf/reg-sub
 :settings/change-password-current-step
 :<- [:settings/change-password]
 :current-step)
 ```

Reply:
no, this I believe does not work, we tried that before, it's because you will get a second argument (the event name or whatever), which means that it will default to it if what it finds is nil, we tried that in the past and got bitten.

`(:blah {:not-present 1} :something) -> :something` that's what you will hit if `change-password?` does not have that key (so I would avoid, even if we think the key will always go there, as it looks like a very natural pattern, but breaks unexpectedly)